netlify.toml

```diff
@@ -0,0 +1,17 @@
+# This configuration is for the standalone version of blockbuilder-search.
+# Used with netlify-cli tool
+# https://www.netlify.com/docs/netlify-toml-reference/
```

Comment:
> separate the Netlify portion of this PR into its own change,

@hydrosquall could you separate this out into it's own PR?  I'd like to get the parcel build local part of this PR working and in, and then take a look at the netlify config.  In theory we should be able to add it too, I just want to test one thing at a time 😄